.vscode/settings.json

```diff
@@ -20,7 +22,9 @@
     "repr",
     "reprompt",
     "reqwest",
+    "rotateable",
     "schemars",
+    "spki",
```

Comment:
suggestion (non-blocking): We seem to be a bit inconsistent on this topic but shouldn't it be uppercase `SPKI` since it's an abbreviation? Same should probably go for things like `DER`

Reply:
Updated.